sys/net/gnrc/transport_layer/tcp/gnrc_tcp_eventloop.c

```diff
@@ -33,6 +33,8 @@
 #define ENABLE_DEBUG (0)
 #include "debug.h"
 
+msg_t _eventloop_msg_queue[TCP_EVENTLOOP_MSG_QUEUE_SIZE];
```

Comment:
does static in this context mean, that _eventloop_msg_queue is only visible in this translation Unit (same behavior as with static functions)?